### ISSUE TITLE
Translate the handbook to Chinese

### ISSUE THREAD

**Issue by artwalker** [rust-lang/cargo#13151]
### Problem

Hi, I want to translate the great handbook to chinese. What should I do? I appreciate what you do for me.

**Comment by hi-rustin** [comment]
Maybe you can contribute to https://github.com/rust-lang-cn/cargo-cn

**Comment by hi-rustin** [comment]
I think adding an official Chinese translation to the cargo handbook is a difficult task. There are mainly two reasons for this:

1. Most maintainers of the handbook are not proficient in Chinese, which makes it challenging to review and maintain the Chinese translation.
2. MdBook, the tool used to create the handbook, does not support multiple languages, making it difficult to reuse the same tool for tracking and deploying the Chinese translation.


So I think it is better to contribute to [rust-lang-cn/cargo-cn](https://github.com/rust-lang-cn/cargo-cn). Then we don't need to translate it from scratch.

**Comment by artwalker** [comment]
Thanks for your advices.

**Comment by weihanglo** [comment]
Yes [rust-lang-cn/cargo-cn](https://github.com/rust-lang-cn/cargo-cn) seems like a better place for the request, as hi-rustin suggested. Other books under `rust-lang` org are also translated outside by volunteers.

For multilingual support of mdbook, please see <https://github.com/rust-lang/mdBook/issues/5>, or other 3rd party mdbook plugin like [google/mdbook-i18n-helpers](https://github.com/google/mdbook-i18n-helpers/).

I wonder if we should have a place to show different translation, just like TRPL does.

* https://doc.rust-lang.org/book/appendix-06-translation.html?highlight=transla#appendix-f-translations-of-the-book
* https://github.com/rust-lang/book/issues?q=is%3Aopen+is%3Aissue+label%3ATranslations

**Comment by artwalker** [comment]
Thank you.

**Comment by artwalker** [comment]
I've created a repository [here](https://github.com/artwalker/cargo-cn) aimed at translating the official documentation into Chinese. Your contributions and suggestions are highly welcomed. Feel free to check it out and join in!

Thanks,
Ethan Wang
